src/service-object.ts

```diff
@@ -42,8 +42,13 @@ export interface Interceptor {
   request(opts: r.Options): DecorateRequestOptions;
 }
 
+// tslint:disable-next-line:no-any
+export type GetMetadataOptions = any;
```

Comment:
I think maybe it would be better to use `object` than `any` here. I think it is safe to assume that we'll only ever allow objects to be passed in (as opposed to strings, numbers, etc.).